lib/root_loader.rb

```diff
@@ -23,19 +25,45 @@ def load account, filename
         end
 
         root_policy_resource = Loader::Types.find_or_create_root_policy(account)
+        policy_version = policy_version(role: admin, policy: root_policy_resource, filename: filename)
+
+        accepted_roles = accepted_roles(policy_version)
+        create_roles(accepted_roles)
```

Comment:
To avoid confusion with the method name, we should use something diff for the local var name:
```ruby
        acc_roles = accepted_roles(policy_version)
        create_roles(acc_roles)
```